front-end/src/components/player-page/header.tsx

```diff
@@ -11,33 +11,35 @@ import LoadingButton from "../loading-button/LoadingButton";
 
 interface Props {
   playerOverview: PlayerOverviewResponse | undefined;
+  onUpdate: () => void;
 }
 
-export default function PlayerPageHeader({playerOverview}: Props)
+export default function PlayerPageHeader(playerOverviewProps: Props)
```

Comment:
Bruh. Why

Reply:
I had to update the props to pass an update function so this needed to be changed